mmseg/models/backbones/ddrnet.py

```diff
@@ -0,0 +1,313 @@
+import math
+import torch
+import numpy as np 
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn import init
+from ..builder import BACKBONES, build_backbone
+from mmcv.runner import BaseModule
+from mmcv.cnn import ConvModule, build_norm_layer
+
+class BasicBlock(nn.Module):
```

Comment:
```suggestion
class BasicBlock(BaseModule):
```
`from mmcv.runner import BaseModule`